From aba90ded077d9f705de41e70dc999ffd062f42f1 Mon Sep 17 00:00:00 2001
From: jiblime <47689567+jiblime@users.noreply.github.com>
Date: Sat, 12 Oct 2019 17:27:21 -0700
Subject: [PATCH] Added KERNEL_MODULES variable to the Makefile Without it,
 systems with kernels that don't use Ubuntu's or Fedora's kernel locations
 will default to '/build'.

---
 Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile b/Makefile
index eafd166..9f36841 100644
--- a/Makefile
+++ b/Makefile
@@ -2,6 +2,8 @@ VERSION         := 0.1.4
 TARGET          := $(shell uname -r)
 DKMS_ROOT_PATH  := /usr/src/zenpower-$(VERSION)
 
+KERNEL_MODULES	:= /lib/modules/$(TARGET)
+
 ifneq ("","$(wildcard /usr/src/linux-headers-$(TARGET)/*)")
 # Ubuntu
 KERNEL_BUILD	:= /usr/src/linux-headers-$(TARGET)
-- 
2.23.0

